packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -168,4 +175,12 @@ const styles = StyleSheet.create({
         gridTemplateRows: "repeat(4, 1fr)",
         backgroundColor: "#DBDCDD",
     },
+    fractionsGrid: {
+        display: "grid",
+        gridTemplateColumns: "repeat(5, 1fr)",
+        gridTemplateRows: "repeat(4, 1fr)",
+        backgroundColor: "#DBDCDD",
+        maxHeight: 200,
+        width: "100%",
```

Comment:
Just a question: why do you think we need this for fractions, but not expression?